src/test/java/com/microsoft/sqlserver/jdbc/fmtOnly/ParameterMetaDataTest.java

```diff
@@ -0,0 +1,151 @@
+                "DELETE " + tableName + " WHERE cFloat >= ? AND ? <= cInt",
+                "DELETE FROM " + tableName + " WHERE cReal BETWEEN ? AND ?;",
+                "DELETE " + tableName + " WHERE cFloat IN (SELECT cFloat FROM " + tableName + " WHERE cReal = ?)",
+                "DELETE " + tableName + " WHERE cFloat IN (SELECT cFloat FROM " + tableName + " t WHERE t.cReal = ?)",
+                "UPDATE TOP (10) " + tableName + " SET cInt = cInt + ?;",
+                "UPDATE TOP (10) " + tableName + " SET cInt = cInt + ? FROM (SELECT TOP 10 cFloat,cReal,cNvarchar FROM "
+                        + tableName + " ORDER BY cFloat ASC) AS b WHERE b.cReal > ? AND b.cNvarchar LIKE ?",
+                "WITH t1(cInt,cReal,cVarchar,cMoney) AS (SELECT cInt,cReal,cVarchar,cMoney FROM " + tableName
+                        + ") UPDATE a SET a.cInt = ? * a.cInt FROM " + tableName
+                        + " AS a JOIN t1 AS b ON a.cInt = b.cInt WHERE b.cInt = ?",
+                "SELECT cInt FROM " + tableName + " WHERE ? = (cInt + (3 - 5))",
+                "SELECT cInt FROM " + tableName + " WHERE (cInt + (3 - 5)) = ?",
+                "SELECT cInt FROM " + tableName + " WHERE " + tableName + ".[cInt] = ?",
+                "SELECT cInt FROM " + tableName + " WHERE ? = " + tableName + ".[cInt]",
+                "WITH t1(cInt) AS (SELECT 1), t2(cInt) AS (SELECT 2) SELECT * FROM t1 JOIN t2 ON [t1].\"cInt\" = \"t2\".[cInt] WHERE \"t1\".[cInt] = [t2].\"cInt\" + ?",
+                "INSERT INTO " + tableName + "(cInt,cFloat) SELECT 1,1.5 WHERE 1 > ?",
+                "WITH t1(cInt) AS (SELECT 1), t2(cInt) AS (SELECT 2), t3(cInt) AS (SELECT 3) SELECT * FROM t1,t2,t3 WHERE t1.cInt >= ?",
+                "SELECT (1),2,[cInt],\"cFloat\" FROM " + tableName + " WHERE cNvarchar LIKE ?",
+                "WITH t1(cInt) AS (SELECT 1) SELECT * FROM \"t1\"");
+        l.forEach(this::compareFmtAndSp);
+    }
+
+    @Test
+    public void noStoredProcTest() throws SQLException {
+        List<String> l = Arrays.asList("SELECT cMoney FROM " + tableName + " WHERE cMoney > $?",
+                "SELECT cVarchar FROM " + tableName + " WHERE " + tableName + ".[cVarchar] NOT LIKE ?",
+                "SELECT cVarchar FROM " + tableName + " WHERE ? NOT LIKE " + tableName + ".[cVarchar]",
+                "INSERT " + tableName + "(cInt) VALUES((1+?)),(3*(?+3))", "SELECT ?");
+        l.forEach(this::executeFmt);
+    }
+
+    @Test
+    public void exceptionTest() throws SQLException {
+        executeInvalidFmt("SELECT FROM OPENQUERY INVALID TSQL", "R_invalidOpenqueryCall");
+        executeInvalidFmt("INSERT INTO OPENXML INVALID TSQL VALUES (?,?,?)", "R_invalidOpenqueryCall");
+        executeInvalidFmt("WITH INVALID_CTE AS SELECT * FROM FOO", "R_invalidCTEFormat");
+    }
+
+    private void executeFmt(String userSQL) {
+        try (Connection c = DriverManager.getConnection(AbstractTest.connectionString + ";useFmtOnly=true;");

```

Comment:
Use PrepUtil wrapper everywhere for new connection with custom connection string for consistency with other Junit tests.